common/metrics/defs.go

```diff
@@ -119,10 +119,13 @@ var GoRuntimeMetrics = map[MetricName]MetricType{
 
 // Scope enum
 const (
+	// -- Operation scopes for ZoneFailoverManager (common) --
+	ZoneFailoverMgrScope = iota
+
```

Comment:
It's strange you prepend to an enum instead of append to the bottom. Is there a specific reason?